Tests/E2E/helper/backendHandleSaveHelper.mjs

```diff
@@ -8,7 +8,6 @@ export default (function () {
 
             await Promise.all([
                 page.waitForResponse(/.*PaypalUnifiedSettings.*/),
-                page.waitForResponse(/.*PaypalUnifiedExpressSettings.*/),
```

Comment:
why did you removed that? :thinking:

Reply:
Because of errors in the local run in the pipeline. And this make no problems. Why not delete code :+1: